src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure {
```

Comment:
The class RightTriangle should not start with an empty line. Please remove the empty line above the class declaration.